### PR TITLE
fix: replace laravel helper by plain php

### PR DESCRIPTION
**Pull request by chrfritsch**

### CODE REVIEW

File: src/Generators/OneImage.php [hunk 1]
```diff
@@ -49,6 +49,6 @@ protected function process()
         $width = $this->file->getWidth() - $this->file->getPadding();
         $height = $this->file->getHeight() - $this->file->getPadding();
 
-        $this->images = collect([$this->images->first()->fit($width, $height)]);
+        $this->images = [[$this->images[0]->fit($width, $height)]];
```
Comment (tzsk) [inline src/Generators/OneImage.php:52]: Should not be nested.

Comment (tzsk) [reply src/Generators/OneImage.php]: Yup. Now it's passing.

**Comment by chrfritsch** [comment]
I think phpcs is not running because of the filename php-cs-fixer.yml.
I experienced the same issue on Github Actions on another project.

**Comment by chrfritsch** [comment]
Or maybe because it only runs on push and not on pull_request

**Comment by tzsk** [comment]
Yes you can add pull_request in the array to fix it.

**Comment by chrfritsch** [comment]
I don't think it will work, because you use the SECRET var in the workflow. Secrets can not be accessed in pull requests

**Comment by chrfritsch** [comment]
So I think you have to use your admin powers to merge this

**Comment by tzsk** [comment]
Sorry. Closing due to https://github.com/tzsk/collage/pull/77

**Pull request closed without merging**